src/network.cc

```diff
@@ -29,6 +33,17 @@ QTcpSocket *tcpSocket =
 
   tcpSocket->connectToHost(ip, port);
 
+  QTime timer;
```

Comment:
@pestOO waitForConnected has already used there, but to avoid locking on waiting I added timer